### PR TITLE
Make company field always an array

### PR DESCRIPTION
**Pull request by brescia123**
As described in #165 I think that `company` field should be always an array to make easier for clients to consume the JSON.

### CODE REVIEW

File: pybikes/bixi.py [hunk 1]
```diff
@@ -168,22 +168,22 @@ def from_json(data):
...
+            "installed": "true",
+            "locked": "false",
+            "installDate": "1313724600000",
+            "removalDate": {  },
+            "temporary": "false",
+            "public": "true",
+            "nbBikes": "15",
+            "nbEmptyDocks": "8",
+            "latestUpdateTime": "1375592453128"
         }
```
Comment (eskerda) [inline pybikes/bixi.py]: This part of the commit is not necessary, unless you do not mind appeariong on the `git blame` for it

**Comment by eskerda** [comment]
Looks good. We should probably add some tests so new systems conform this rule, will do that on another PR

File: pybikes/bixi.py (hunk 1 above)
Comment (brescia123) [inline pybikes/bixi.py]: Of course, that was a mistake! 😅

**Comment by eskerda** [comment]
Looks good, thanks!

**Pull request merged**